test-kuttl/kubeconfig

```diff
@@ -0,0 +1,16 @@
+clusters:
```

Comment:
Did you mean to commit this?

Reply:
Doh! I think this happens if you `test-e2e` and the test fails before it can clean up. I'll add this to gitignore.